src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchElementViewModel.cs

```diff
@@ -40,28 +79,46 @@ public class PackageManagerSearchElementViewModel : BrowserItemViewModel, IEquat
         public PackageManagerSearchElementViewModel(PackageManagerSearchElement element, bool canLogin, bool install, bool isEnabledForInstall = true) 
             : base(element)
         {
-            this.Model = element;
+            this.SearchElementModel = element;
             CanInstall = install;
             IsEnabledForInstall = isEnabledForInstall;
 
-            this.SelectedVersion = this.Model.LatestVersion;
+            this.SelectedVersion = this.SearchElementModel.LatestVersion;
 
-            this.ToggleIsExpandedCommand = new DelegateCommand(() => this.Model.IsExpanded = !this.Model.IsExpanded);
+            this.ToggleIsExpandedCommand = new DelegateCommand(() => this.SearchElementModel.IsExpanded = !this.SearchElementModel.IsExpanded);
 
             this.DownloadLatestCommand = new DelegateCommand(
-                () => OnRequestDownload(Model.Header.versions.First(x => x.version.Equals(SelectedVersion)), false),
-                () => !Model.IsDeprecated && CanInstall);
-            this.DownloadLatestToCustomPathCommand = new DelegateCommand(() => OnRequestDownload(Model.Header.versions.First(x => x.version.Equals(SelectedVersion)), true));
+                () => OnRequestDownload(SearchElementModel.Header.versions.First(x => x.version.Equals(SelectedVersion)), false),
+                () => !SearchElementModel.IsDeprecated && CanInstall);
+            this.DownloadLatestToCustomPathCommand = new DelegateCommand(() => OnRequestDownload(SearchElementModel.Header.versions.First(x => x.version.Equals(SelectedVersion)), true));
 
-            this.UpvoteCommand = new DelegateCommand(Model.Upvote, () => canLogin);
+            this.UpvoteCommand = new DelegateCommand(SearchElementModel.Upvote, () => canLogin);
 
             // TODO: Remove the initialization of the UI command in Dynamo 3.0
-            this.DownvoteCommand = new DelegateCommand(Model.Downvote, () => canLogin);
+            this.DownvoteCommand = new DelegateCommand(SearchElementModel.Downvote, () => canLogin);
 
             this.VisitSiteCommand =
-                new DelegateCommand(() => GoToUrl(FormatUrl(Model.SiteUrl)), () => !String.IsNullOrEmpty(Model.SiteUrl));
+                new DelegateCommand(() => GoToUrl(FormatUrl(SearchElementModel.SiteUrl)), () => !String.IsNullOrEmpty(SearchElementModel.SiteUrl));
             this.VisitRepositoryCommand =
-                new DelegateCommand(() => GoToUrl(FormatUrl(Model.RepositoryUrl)), () => !String.IsNullOrEmpty(Model.RepositoryUrl));
+                new DelegateCommand(() => GoToUrl(FormatUrl(SearchElementModel.RepositoryUrl)), () => !String.IsNullOrEmpty(SearchElementModel.RepositoryUrl));
+
+            SetModelProperties();
+        }
+
+        /// <summary>
+        /// Sets the VM properties to be bound to
+        /// </summary>
+        private void SetModelProperties()
```

Comment:
this method name seems backwards to me, isn't this setting ViewModel properties from the model?

Reply:
`SetViewModelProperties` sounds more suitable, I will change that!